smart-contracts/migrations/2_next.js

```diff
@@ -16,55 +16,48 @@ module.exports = function(deployer, network, accounts) {
   let operator;
   let owner;
   let initialValidators = [];
-  let initialPowers = [];
-  let consensusThreshold;
-
-  // Input validation for general env variables
-  if (process.env.CONSENSUS_THRESHOLD.length === 0) {
-    return console.error(
-      "Must provide consensus threshold as environment variable."
-    );
-  }
-  consensusThreshold = process.env.CONSENSUS_THRESHOLD;
+  let consensusThreshold = (process.env.CONSENSUS_THRESHOLD === 0 ? process.env.CONSENSUS_THRESHOLD:70)
```

Comment:
I think the ternary operator here needs to be reversed. If the consensus threshold is 0, we should default to 70, otherwise use the consensus threshold from the env var.

`  let consensusThreshold = (process.env.CONSENSUS_THRESHOLD === 0 ? 70 : process.env.CONSENSUS_THRESHOLD)
`